### PR TITLE
WFLY-13176 Add jboss-ejb-client_1_3.xsd schema

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
https://issues.redhat.com/browse/WFLY-13176

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
I have added a jboss-ejb-client_1_3.xsd schema. Please review and let me know what you think ! :)

File: ejb3/src/main/resources/schema/jboss-ejb-client_1_3.xsd [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+	<xsd:complexType name="profileType">
+		<xsd:attribute name="name" type="xsd:string" use="required">
+			<xsd:annotation>
+				<xsd:documentation>
+					The name of the profile.
+				</xsd:documentation>
+			</xsd:annotation>
+		</xsd:attribute>
+	</xsd:complexType>
+
```
Comment (chengfang) [inline ejb3/src/main/resources/schema/jboss-ejb-client_1_3.xsd]: need to fix the indentation, eg. replace tab with space consistent with other elements.

File: ejb3/src/main/resources/schema/jboss-ejb-client_1_3.xsd [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+                    <xsd:documentation>
+                        Cluster configurations applicable for this client context
+                    </xsd:documentation>
+                </xsd:annotation>
+            </xsd:element>
+            <xsd:element name="profile" type="profileType" minOccurs="0">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        Cluster configurations applicable for this client context
+                    </xsd:documentation>
```
Comment (chengfang) [inline ejb3/src/main/resources/schema/jboss-ejb-client_1_3.xsd]: need to change documentation to reflect the purpose of the new element.  Probably something like `Remoten connection profile selected for this client context.`

**Comment by chengfang** [comment]
@tadamski please review

`profile` element and `profileType` element definition are the only additon compared to jboss-ejb-client 1.2.

**Review by chengfang — APPROVED** [review]

File: ejb3/src/main/resources/schema/jboss-ejb-client_1_3.xsd [hunk 3]
```diff
@@ -0,0 +1,330 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2012, Red Hat, Inc., and individual contributors
```
Comment (rhusar) [inline ejb3/src/main/resources/schema/jboss-ejb-client_1_3.xsd]: My watch says its 2020.

**Review by rhusar — CHANGES_REQUESTED** [review]

File: ejb3/src/main/resources/schema/jboss-ejb-client_1_3.xsd (hunk 3 above)
Comment (RanabirChakraborty) [reply ejb3/src/main/resources/schema/jboss-ejb-client_1_3.xsd]: Previously I thought I would change it to 2020 but I was not sure how the copyright think works.You can check now, I have made the changes. :)

**Review by tadamski — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks!

**Pull request merged**